src/certificate-authority.ts

```diff
@@ -141,13 +108,37 @@ export async function ensureCACertReadable(options: Options = {}): Promise<void>
    * has no read permissions either way, openssl will fail and that means we
    * have to fix it
    */
-  const caFileContents = await currentPlatform.readProtectedFile(rootCACertPath);
-  await currentPlatform.deleteProtectedFile(rootCACertPath);
-  writeFile(rootCACertPath, caFileContents);
+  try {
+    const caFileContents = await currentPlatform.readProtectedFile(rootCACertPath);
+    currentPlatform.deleteProtectedFiles(rootCACertPath);
+    writeFile(rootCACertPath, caFileContents);
+  } catch (e) {
+    return installCertificateAuthority(options);
+  }
   
   // double check that we have a live one
   const remainingErrors = certErrors();
   if (remainingErrors) {
     return installCertificateAuthority(options);
   }
 }
```

Comment:
I like this; it works well.  If the CA has to be rebuilt, would it be possible to drop a message in the console?  The reason I ask is because I get popups in Windows when it wants to remove the trust.  It's a big deal, if you know what's happening, and why; some people might get thrown off by it popping up and asking them to delete stuff when they're trying to run a development server, though.

Reply:
I added a console warning for this! Please do open a followup PR if the wording isn't great. I'm really more of a good speller than a good writer.